queue/execution.go

```diff
@@ -72,7 +75,7 @@ func (eq *ExecutionQueue) Add(exec Executor) {
 		eq.queues[repository] = newPriorityQueue()
 	}
 	eq.queues[repository].add(exec)
-	eq.incQueueGauge(exec.GetJobNamespace(), exec.GetJobType())
+	eq.incQueueGauge(exec.GetJobNamespace(), string(exec.GetJobType()))
```

Comment:
I think conversion to string should happen in the `incQueueGauge` itself and not in the caller